File: BarberShop.Service/Models/OrderInfo.cs
using System;
using System.Collections.Generic;

namespace BarberShop.Service.Models
{
    public class OrderInfo
    {
        public int Id { get; set; }
        public Customer CustomerInfo { get; set; }
        public Employee EmployeeInfo { get; set; }
        public ShopAddress ShopAddressInfo { get; set; }
        public Payment PaymentInfo { get; set; }
        public List<ServiceInfo> ServicesInfo { get; set; }
        public DateTime OrderDate { get; set; }
    }
}

File: BarberShop.App/src/app/models/OrderInfo.ts
import { Customer } from "./Customer";
import { Employee } from "./Employee";
import { Payment } from "./Payment";
import { ServiceInfo } from "./ServiceInfo";
import { ShopAddress } from "./ShopAddress";

export class OrderInfo{
    id: number;
    customerInfo: Customer;
    employeeInfo: Employee;
    shopAddressInfo: ShopAddress;
    paymentInfo: Payment;
    servicesInfo: ServiceInfo[];
    orderDate: Date;
    hide: boolean;
}